src/Video.tsx

```diff
@@ -193,9 +193,16 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
       if (!selectedTextTrack) {
         return;
       }
-      const type = typeof selectedTextTrack.value;
-      if (type !== 'number' && type !== 'string') {
-        console.log('invalid type provided to selectedTextTrack');
+      const typeOfValueProp = typeof selectedTextTrack.value;
+      if (
+        typeOfValueProp !== 'number' &&
+        typeOfValueProp !== 'string' &&
+        typeOfValueProp !== 'undefined'
+      ) {
+        console.log(
+          'invalid type provided to selectedTextTrack.value: ',
+          typeOfValueProp,
+        );
```

Comment:
Let's use `console.error` for those

Reply:
Maybe console.warn is enough? As you want!